### PR TITLE
Add Evergreen pre-release

### PR DESCRIPTION
**Pull request by ramsey**
Evergreen is currently in a pre-alpha state, described by the author as "barely usable at this point. Not even that. It’s not beta — it’s not even alpha." Therefore, I am submitting this to the main repo. The author previously created NetNewsWire and MarsEdit, and there is a lot of anticipation for this new project, so I expect it to continue to a version 1 in the near-ish future.

I originally attempted to add a `zap` section, but after removing the user-specific files and then reinstalling, the app did not return to "factory settings." Only after [following these instructions](https://github.com/brentsimmons/Evergreen/blob/master/Technotes/RestoreToFactory.md) could I get it to return to factory settings, and I was unable to run these commands using the `uninstall script:` section. I figured the best approach was to leave all the user-specific files in place for now.

---

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/evergreen.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'evergreen' do
+  version :latest
```
Comment (commitay) [inline Casks/evergreen.rb]: `1.0d30`

File: Casks/evergreen.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  sha256 :no_check
```
Comment (commitay) [inline Casks/evergreen.rb]: Please add the `sha256`

File: Casks/evergreen.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+  url 'https://ranchero.com/downloads/Evergreen-latest.zip'
```
Comment (commitay) [inline Casks/evergreen.rb]: `"https://ranchero.com/downloads/Evergreen#{version}.zip"`

**Review by commitay — CHANGES_REQUESTED** [review]

File: Casks/evergreen.rb (hunk 3 above)
Comment (ramsey) [reply Casks/evergreen.rb]: Thanks for catching these. I should have read the contents of the appcast. :smile:

**Comment by commitay** [comment]
> ...  the app did not return to "factory settings."

`zap` isn't a "reset" function, it's only for file cleanup. You can the `application support` folder back if you want but we don't run things like `defaults delete` and `killall cfprefsd` as part of `uninstall` or `zap`.

**Comment by ramsey** [comment]
I understand that `zap` is not a reset function. That's not what I was trying to use it for. However, the first time you launch the app, it comes pre-configured with a list of feeds. If you modify that list, then uninstall the app with `zap` (removing the Application Support folder), it blows everything away, so the next time you install it, there are no default feeds—or any feeds at all.

I found the full list of user-specific files and preferences to move to trash with `zap`, but I could only "reset" the app to the clean install settings by using `defaults delete` and then `killall cfprefsd`.

**Comment by ramsey** [comment]
It looks like the Travis CI build is failing because Travis doesn't support High Sierra yet.

**Review by commitay — APPROVED** [review]
Install verified locally.

**Review by victorpopkov — APPROVED** [review]

**Pull request merged**